docs/contribute/modules/index.md

```diff
@@ -99,21 +99,31 @@ repository example-contributor/contributor-theorist
 
 ### Install the "parent" package in development mode
 
-Install this in an environment using your chosen package manager. In this example, we use pip and virtualenv.
-
-First, install:
+!!! success
+    We recommend setting up your development environment using a manager like `venv`, which creates isolated python 
+    environments. Other environment managers, like 
+    [virtualenv](https://virtualenv.pypa.io/en/latest/),
+    [pipenv](https://pipenv.pypa.io/en/latest/),
+    [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), 
+    [hatch](https://hatch.pypa.io/latest/), 
+    [poetry](https://python-poetry.org), 
+    are available and will likely work, but will have different syntax to the syntax shown here. 
 
-- python: https://www.python.org/downloads/
-- virtualenv: https://virtualenv.pypa.io/en/latest/installation.html
+Run the following command to create a new virtual environment in the `.venv` directory
 
-Create a new virtual environment:
 ```shell
-virtualenv venv
+python3 -m "venv" ".venv" 
 ```
 
-Activate it:
+!!! hint
+    If you have multiple Python versions installed on your system, it may be necessary to specify the Python version when creating a virtual environment. For example, run the following command to specify the Python 3.8 for the virtual environment. 
```

Comment:
```suggestion
    If you have multiple Python versions installed on your system, it may be necessary to specify the Python version when creating a virtual environment. For example, run the following command to specify Python 3.8 for the virtual environment. 
```